commit b71fae852b2b3ecf45b3428d78ceac3ea8393f53 (refs/changes/73/4573/2)
Author: Rui Loura <rui@joyent.com>
Date:   2018-07-25T20:13:33+00:00 (1 year, 2 months ago)
    
    TRITON-508 config-agent autometadata should contain an array of IPs for each nic (bump version)
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index 5a00fc0..701079c 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "config-agent",
     "description": "SmartDataCenter Config Agent",
-    "version": "1.7.1",
+    "version": "1.8.0",
     "author": "Joyent (joyent.com)",
     "dependencies": {
         "assert-plus": "0.1.5",
